### PR TITLE
🌐 Add Spanish translation for `advanced/path-operation-advanced-configuration.md`

### PR DESCRIPTION
**Pull request by jfunez**

### CODE REVIEW

File: docs/es/docs/advanced/path-operation-advanced-configuration.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Si tu quieres usar tus nombres de funciones de API como `operationId`s, puedes iterar sobre todos ellos y sobrescribir `operation_id` de cada *operación de camino* usando su `APIRoute.name`.
+
+Deberías hacerlo después de adicionar todas tus *operaciones de camino*.
+
+```Python hl_lines="2 12 13 14 15 16 17 18 19 20 21 24"
+{!../../../docs_src/path_operation_advanced_configuration/tutorial002.py!}
+```
+
+!!! tip
+    Si llama manualmente a `app.openapi()`, debe actualizar el `operationId`s antes de eso.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/path-operation-advanced-configuration.md:27]:
```suggestion
    Si llamas manualmente a `app.openapi()`, debes actualizar el `operationId`s antes de hacerlo.
```

File: docs/es/docs/advanced/path-operation-advanced-configuration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+!!! warning
+    Si hace esto, debes asegurarte de que cada una de tus *funciones de operación de camino* tenga un nombre único.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/path-operation-advanced-configuration.md:30]:
```suggestion
    Si haces esto, debes asegurarte de que cada una de tus *path operation  functions* tenga un nombre único.
```

File: docs/es/docs/advanced/path-operation-advanced-configuration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+    Incluso si están en diferentes módulos (archivos Python).
+
+## Excluir de OpenAPI
+
+Para excluir una *operación de camino* del esquema OpenAPI generado (y por tanto del la documentación generada automáticamente), use el parámetro `include_in_schema` y asigne el valor como `False`;
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/path-operation-advanced-configuration.md:36]:
```suggestion
Para excluir una *path operation* del esquema OpenAPI generado (y por tanto del la documentación generada automáticamente), usa el parámetro `include_in_schema` y asigna el valor como `False`;
```

File: docs/es/docs/advanced/path-operation-advanced-configuration.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+```Python hl_lines="6"
+{!../../../docs_src/path_operation_advanced_configuration/tutorial003.py!}
+```
+
+## Descripción Avanzada desde docstring
+
+Puedes limitar las líneas usadas desde la docstring de una *operación de camino* para OpenAPI.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/path-operation-advanced-configuration.md:44]:
```suggestion
Puedes limitar las líneas usadas desde la docstring de una *path operation* para OpenAPI.
```

File: docs/es/docs/advanced/path-operation-advanced-configuration.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Adding an `\f` (an escaped "form feed" character) causes **FastAPI** to truncate the output used for OpenAPI at this point.
```
Comment (mariacamilagl) [inline docs/es/docs/advanced/path-operation-advanced-configuration.md:46]:
```suggestion

```

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thanks for the translation, @jfunez  🚀 

I'm suggesting to leave all FastAPI specific terms, like path operations or path operation functions in English. They should be explained in earlier stages of the docs (like the beginner's tutorial) and this way we maintain consistency.

**Review by mariacamilagl — APPROVED** [review]
Thanks for your effort @jfunez!

**Comment by tiangolo** [comment]
Awesome, thanks @jfunez! :coffee: :cake: 

And thanks @mariacamilagl for the reviews and corrections! :nerd_face: :coffee:

**Pull request merged**